ssc/cmod_cashloan.cpp

```diff
@@ -91,30 +91,29 @@ static var_info vtab_cashloan[] = {
     { SSC_OUTPUT, SSC_NUMBER, "total_cost", "Total installed cost", "$", "", "Financial Metrics", "*", "", "" },
 		
 
-	{ SSC_OUTPUT,        SSC_ARRAY,      "cf_energy_net",      "Energy",                  "kWh",            "",                      "Cash Flow",      "*",                     "LENGTH_EQUAL=cf_length",                "" },
-	{ SSC_OUTPUT,        SSC_ARRAY,      "cf_energy_value",      "Value of electricity savings",                  "$",            "",                      "Cash Flow",      "*",                     "LENGTH_EQUAL=cf_length",                "" },
+    { SSC_OUTPUT,       SSC_ARRAY,      "cf_energy_net",            "Electricity to grid net",       "kWh", "", "Cash Flow Electricity", "*", "LENGTH_EQUAL=cf_length", "" },
```

Comment:
These labels don't translate well to cashloan. Here is a PV-Battery residential case with grid charging turned on:

![image](https://user-images.githubusercontent.com/5530592/153267916-ca33a22f-a240-4ca1-be7c-a261f33b1ee8.png)

First, note that "net" and "to grid" are the same number, despite a sizeable number of kWh from grid. It's unclear how much of this energy actually makes it to the grid, but the monthly powerflow outputs imply roughly half:

![image](https://user-images.githubusercontent.com/5530592/153268916-62e387d9-aaa9-4049-a5a3-ecad8437b725.png)

The metrics chart appears to have the correct net number for year one:

![image](https://user-images.githubusercontent.com/5530592/153269054-d92756f0-eb03-4413-acba-7392a541cd4a.png)

Second, the "from grid" label needs to discuss the system somehow, as a sizeable amount of grid power is going to load.